be/src/exec/pipeline/analysis/analytic_sink_operator.cpp

```diff
@@ -33,9 +33,15 @@ Status AnalyticSinkOperator::prepare(RuntimeState* state) {
             _process_by_partition_if_necessary = &AnalyticSinkOperator::_process_by_partition_if_necessary_for_other;
             _process_by_partition = &AnalyticSinkOperator::_process_by_partition_for_unbounded_frame;
         } else if (!window.__isset.window_start && window.window_end.type == TAnalyticWindowBoundaryType::CURRENT_ROW) {
-            _process_by_partition_if_necessary =
-                    &AnalyticSinkOperator::
-                            _process_by_partition_if_necessary_for_rows_between_unbounded_preceding_and_following;
+            if (!_analytor->need_partition_boundary_for_unbounded_preceding_rows_frame()) {
```

Comment:
need partition boundary=false  --> _process_by_partition_if_necessary_for_rows_between_unbounded_preceding_and_following ?

Reply:
Yes, when it is `ROWS BETWEEN UNBOUNDED PRECEDING AND CURRENT ROWS` and needn't partition boundary, it can use the new `_process_by_partition_if_necessary_for_rows_between_unbounded_preceding_and_following`

Reply:
My thinking should be the opposite